### PR TITLE
Fix conditional include of driver module

### PR DESCRIPTION
**Pull request by vchong**
Signed-off-by: Victor Chong <victor.chong@linaro.org>
Tested-by: Victor Chong <victor.chong@linaro.org> (QEMU with CONFIG_OPTEE=m)

### CODE REVIEW

**Comment by jbech-linaro** [comment]
Seems to be working fine on QEMU, thanks for the quick fix Victor.
```
Reviewed-by: Joakim Bech <joakim.bech@linaro.org>
Tested-by: Joakim Bech <joakim.bech@linaro.org> (QEMU)
```

**Comment by vchong** [comment]
@pascal-brand-st-dev I added Joakim's tags. Please lemme know if you need another or ok to merge. Thanks!

File: hikey.mk [hunk 1]
```diff
@@ -273,12 +273,12 @@ filelist-all: busybox
 	find $(OPTEE_TEST_OUT_PATH) -type f -name "xtest" | sed 's/\(.*\)/file \/bin\/xtest \1 755 0 0/g' >> $(GEN_ROOTFS_PATH)/filelist-all.txt; \
 	find $(OPTEE_TEST_OUT_PATH) -name "*.ta" | \
 		sed 's/\(.*\)\/\(.*\)/file \/lib\/optee_armtz\/\2 \1\/\2 444 0 0/g' >> $(GEN_ROOTFS_PATH)/filelist-all.txt
-	@if [ -e $(OPTEE_GENDRV_MODULE) ]; then
-		@echo "# OP-TEE device" >> $(GEN_ROOTFS_PATH)/filelist-all.txt
-		@echo "dir /lib/modules 755 0 0" >> $(GEN_ROOTFS_PATH)/filelist-all.txt
-		@echo "dir /lib/modules/$(call KERNEL_VERSION) 755 0 0" >> $(GEN_ROOTFS_PATH)/filelist-all.txt
-		@echo "file /lib/modules/$(call KERNEL_VERSION)/optee.ko $(OPTEE_GENDRV_MODULE) 755 0 0" >> $(GEN_ROOTFS_PATH)/filelist-all.txt
-	@fi
+	if [ -e $(OPTEE_GENDRV_MODULE) ]; then \
```
Comment (pascal-brand-st-dev) [inline hikey.mk]: s/if/@if

File: hikey.mk [hunk 2]
```diff
@@ -273,12 +273,12 @@ filelist-all: busybox
...
+		echo "# OP-TEE device" >> $(GEN_ROOTFS_PATH)/filelist-all.txt; \
+		echo "dir /lib/modules 755 0 0" >> $(GEN_ROOTFS_PATH)/filelist-all.txt; \
+		echo "dir /lib/modules/$(call KERNEL_VERSION) 755 0 0" >> $(GEN_ROOTFS_PATH)/filelist-all.txt; \
+		@echo "file /lib/modules/$(call KERNEL_VERSION)/optee.ko $(OPTEE_GENDRV_MODULE) 755 0 0" >> $(GEN_ROOTFS_PATH)/filelist-all.txt; \
```
Comment (pascal-brand-st-dev) [inline hikey.mk]: s/@echo/echo

File: hikey.mk (hunk 1 above)
Comment (vchong) [inline hikey.mk]: Updated

File: hikey.mk (hunk 2 above)
Comment (vchong) [inline hikey.mk]: Sigh.. looks like i'm going to need glasses.
Thanks for catching this.

**Comment by vchong** [comment]
Comments addressed.

**Comment by pascal-brand-st-dev** [comment]
Sorry for the late notice: mediatek.mk is not updated.

**Comment by vchong** [comment]
Thanks! mediatek.mk updated.

**Pull request merged**